File: linux/ardium_source/presentation/05_Strings/strings.m
// Topic: Strings
// Concatenation and Printing

#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        NSString *s1 = @"Hello";
        NSString *s2 = @" World";
        
        NSString *s3 = [s1 stringByAppendingString:s2];
        
        NSLog(@"Result: %@", s3);
    }
    return 0;
}

File: linux/ardium_source/presentation/05_Strings/strings.rs
// Topic: Strings
// Concatenation and Printing

fn main() {
    let s1 = "Hello".to_string();
    let s2 = " World";
    
    // Rust requires ownership transfer or cloning for +
    let s3 = s1 + s2;
    
    println!("Result: {}", s3);
}
